### DISCUSSION TITLE
Lista para IPTV apple

### DISCUSSION THREAD

**Discussion by tilfreire** [Ramys/Iptv-Brasil-2026#66] [Perguntas e respostas]
Oi, você tem alguma lista que eu possa usar para assistir canais do Brasil no app IPTV em iphone?

**Comment by Luthem01** [comment]
Olá eu tenho. O aplicativo que eu indico para utilizar no iPhone é o IPTV Smarters.

**Comment by jogosjk** [comment]
manda uma lista top atualizada completa, e também algum programa que captura so listas que funciona ?. -
